### PR TITLE
Show Powerups in Progress Bar

### PR DESCRIPTION
**Pull request by kevinsun-dev**
## Overview

Added Textures for Opponent Progress Powerups

## Next Steps

Add Powerup Icons to Tutorial Page

## Screenshots

<details>

  <summary>Screenshot</summary>
  
Screen Shot 2021-05-08 at 5 19 14 PM

</details>

### CODE REVIEW

File: source/CIGameScene.cpp [hunk 1]
```diff
@@ -117,7 +117,12 @@ bool GameScene::init(const std::shared_ptr<cugl::AssetManager>& assets,
     auto unlockedTexture = _assets->get<Texture>("unlockedOuterRing");
     auto lockedTexture = _assets->get<Texture>("lockedOuterRing");
     auto planetProgressTexture = _assets->get<Texture>("playerProgress");
-    _planet->setTextures(coreTexture, ringTexture, unlockedTexture, lockedTexture, planetProgressTexture);
+    std::vector<std::shared_ptr<cugl::Texture>> powerupTexture;
```
Comment (henryz2000) [inline source/CIGameScene.cpp:120]: NIT: `powerupTextures`

**Review by henryz2000 — APPROVED** [review]
we should probably explain somewhere what these powerups are/what they do

**Review by JackieW001 — APPROVED** [review]
Looks good!

File: source/CIGameScene.cpp (hunk 1 above)
Comment (kevinsun-dev) [reply source/CIGameScene.cpp]: Fixed!

**Pull request merged**